gcp_variant_transforms/libs/variant_partition.py

```diff
@@ -90,3 +289,29 @@ def _get_auto_partition(self, reference_name):
       # Save partition in _reference_name_to_partition dict for future lookups
       self._reference_name_to_partition_map[reference_name] = partition
     return partition
+
+  def is_default_partition_absent(self):
```

Comment:
It's more readable to have a positive method name rather than double-negative so `has_residual_partition` is a better name.

Reply:
done.